It appears that Microsoft has modified the MSI file some time before 11th of June 2021, and it now has a different hash.
I've compared the files from old and new version of the MSI file and none of its contents are different (most importantly,
the firmware we utilise is the exact same), so the simplest fix is just to update the hash.

diff --git a/kinect_fetch_fw b/kinect_fetch_fw
index 91b1289..d058b70 100755
--- a/kinect_fetch_fw
+++ b/kinect_fetch_fw
@@ -13,7 +13,7 @@
 set -e
 
 SDK_URL=${SDK_URL:-"http://download.microsoft.com/download/F/9/9/F99791F2-D5BE-478A-B77A-830AD14950C3/KinectSDK-v1.0-beta2-x86.msi"}
-SDK_MD5="40764fe9e00911bda5095e5be777e311"
+SDK_MD5="945806927702b2c47c32125ab9a80344"
 
 [ $# -lt 1 ] && { echo "usage: $(basename "$0") <firmware destdir>" 1>&2; exit 1; }
 FW_DESTDIR=$(readlink -m $1)
